documentation/asciidoc/computers/raspberry-pi/boot-eeprom.adoc

```diff
@@ -13,10 +19,13 @@ If an error occurs during boot, then an xref:configuration.adoc#led-warning-flas
 
 There are multiple ways to update the bootloader of your Raspberry Pi.
 
-==== Raspberry Pi 5, Raspberry Pi 4 and Raspberry Pi 400
+==== Raspberry Pi 5, Compute Module 5, Raspberry Pi 500, Raspberry Pi 4 and Raspberry Pi 400
 
 Raspberry Pi OS automatically updates the bootloader for important bug fixes. To manually update the bootloader or change the boot order, use xref:configuration.adoc#raspi-config[raspi-config].
 
+N.B. On Compute Module 4 and CM4S the bootrom is unable to load the `recovery.bin` file from EMMC and automatic bootloader updates are not enabled.
```

Comment:
```suggestion
NOTE: Compute Module 4 and Compute Module 4S do not support automatic bootloader updates because the bootrom cannot load the `recovery.bin` file from eMMC.
```